build.sh

```diff
@@ -43,6 +43,13 @@ case $ARCH in
 		MICROARCH="${ARCH}"
 		ARCH="ppc"
 		;;
+	"rv64"*)
+		# only support riscv64 for now
+		DOCKER_ARCH=riscv64
+		MICROARCH="${ARCH}-20.0"
```

Comment:
```suggestion
		MICROARCH="${ARCH}"
```

Profile suffixes have been dropped upstream: https://github.com/gentoo/releng/commit/adfc9e915ee5b91d894a9f435ecfbece1f158b9e